snapshot/android/snapshot-generator.js

```diff
@@ -22,24 +24,69 @@ module.exports = SnapshotGenerator;
 SnapshotGenerator.MKSNAPSHOT_TOOLS_PATH = path.join(__dirname, "snapshot-generator-tools/mksnapshot");
```

Comment:
Why do you think we need `getAbsolutePath` function? It is as verbose as using `join(__dirname, path)` but I find `join(__dirname, path)` to be more clear.